panel/param.py

```diff
@@ -53,37 +53,34 @@ class Skip(RuntimeError):
 from .pane.base import Pane, ReplacementPane
 from .reactive import Reactive
 from .util import (
-    abbreviated_repr, flatten, full_groupby, fullpath, is_parameterized,
-    param_name, recursive_parameterized, to_async_gen,
+    abbreviated_repr, classproperty, flatten, full_groupby, fullpath,
+    is_parameterized, param_name, recursive_parameterized, to_async_gen,
 )
 from .util.checks import is_dataframe, is_mpl_axes, is_series
 from .viewable import Layoutable, Viewable
-from .widgets import (
```

Comment:
I am fairly confused by inlining this stuff. The `panel.widgets` module is imported by default.

Reply:
This is mostly preparation for the lazy import PR to avoid circular imports.

I can move these changes into that PR or open a new PR with only the necessary changes, whatever is easiest.

Reply:
Would prefer that as I want to do that stuff after this release and carefully consider the approach.